_gitbook/conventions/README.md

```diff
@@ -0,0 +1,6 @@
+# Conventions
+
+Follow these conventions so your code will be more accessible to other developers.
+
+* Use [standard coding style](coding_style.md) so your project will be navigable and readable to others.
```

Comment:
I feel like this is missing a modifier. Something like "Use the standard coding style"?